README.md

```diff
@@ -251,3 +264,56 @@ async function run() {
 
 run();
 ```
+
+### Full example for building a single `_variable.scss` for general WEB usecase.
+
+Create a `.js` for eg: `build-styles.js` file with the contents 
+
+```js
+// Import StyleDictionary and the transforms library
+import StyleDictionary from 'style-dictionary'
+import { registerTransforms, transforms } from '@tokens-studio/sd-transforms'
+
+// Extend StyleDictionary with the configuration file
+const styleDictionary = StyleDictionary.extend('./style-dictionary-config.json')
+
+// Register transforms
+registerTransforms(styleDictionary)
+
+/* Register custom tokens-studio transform group without 
+descriptions being mapped to comments
+and also adding 'name/cti/kebab' for the token names */
+
+styleDictionary.registerTransformGroup({
+  name: 'custom/tokens-studio',
+  transforms: [...transforms, 'name/cti/kebab'].filter(
+    (transform) => transform !== 'ts/descriptionToComment',
+  ),
+})
+
+// Format helpers from StyleDictionary
+const { fileHeader, formattedVariables } = styleDictionary.formatHelpers
+
+// Format Css
+styleDictionary.registerFormat({
+  name: 'myCustomFormat',
+  formatter: function ({ dictionary, file, options }) {
+    const { outputReferences } = options
+    return `${fileHeader({ file })}:root {
+${formattedVariables({ format: 'scss', dictionary, outputReferences })}
+}`
+  },
+})
+
+// Clean web platform before building
+styleDictionary.cleanPlatform('web')
+
+// Build web platform
+styleDictionary.buildPlatform('web')
+```
+
+## To run it 
+
+```sh
+node build-styles.js
+```
```

Comment:
- There is already a full example which is web specific..?
- There is already an example for custom transform group with camel case, although that should probably be kebab case because camel case is the default (more language-agnostic), so this one doesn't add much value to the docs that already exist
- No specification of what the content is of "style-dictionary-config.json"